eessi_bot_software_layer.py

```diff
@@ -78,15 +94,15 @@ def main():
 
     if opts.file:
         event = read_event_from_json(opts.file)
-        log_event(event)
-        handle_event(event)
+        event_info = get_event_info(event)
+        handle_event(event_info)
     elif opts.cron:
         log("Running in cron mode")
     else:
         # Run as web app
-        app = create_app()
-        app.run()
-
+        app = create_app(klass=EESSIBotSoftwareLayer)
+        log("EESSI bot for software layer started!")
+        waitress.serve(app, listen='*:3000')
```

Comment:
This is also something small we can move to PyGHee: this should just become `app.start(port=3000)` or something like that (which avoids having to import `waitress` here, that should be internal to PyGHee)

Reply:
Created issue #9 for this.